src/mock/cards.ts

```diff
@@ -0,0 +1,48 @@
+export type cards = {
+  id: number;
+  imageUrl: string;
+  label: string;
+};
+
+export const messageCards: cards[] = [
+  {
+    id: 1,
+    imageUrl: 'https://t1.kakaocdn.net/kakaogift/resource/card/cheer/card_01.png',
+    label: '축하해요.',
+  },
```

Comment:
이미지 url이 잘못된것 같아요. 과제의 주문하기 [카드 템플릿 목데이터](https://edu.nextstep.camp/s/0eoNzeZS/ls/p9PJBAjn)로 수정해주세요

```suggestion
  {
    id: 904,
    thumbUrl: 'https://t1.daumcdn.net/gift/message-card/template/image/20240124_VGPJE.png',
    imageUrl: 'https://t1.daumcdn.net/gift/message-card/template/image/20240124_NRPQT.gif',
    defaultTextMessage: '축하해요.',
  },
```